src/ORM/Marshaller.php

```diff
@@ -214,6 +214,9 @@ protected function _prepareDataAndOptions($data, $options)
      */
     protected function _marshalAssociation($assoc, $value, $options)
     {
+        if (!is_array($value)) {
+            return;
```

Comment:
because this is browser submitted data. You don't want your server to be hammered with data that triggers exceptions